pkgs/development/python-modules/django-auth-ldap/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, ldap , django_2_2 
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "django-auth-ldap";
+  version = "1.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "72848b3b036d299114be3c6ef38b12f83f6cf1cf1696c5f92e06fe45a1b8e27b";
+  };
+
+  propagatedBuildInputs = [ ldap django_2_2 ]; 
+  checkInputs = [ mock ]; 
+
+  # django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Django LDAP authentication backend";
+    homepage = https://github.com/django-auth-ldap/django-auth-ldap;
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ mmai ];
+    broken = stdenv.isDarwin;
```

does this even support darwin? If it doesn't it would make more sense to set platforms
```suggestion
    platforms = platforms.linux
```

---

fixed